requirements.txt

```diff
@@ -1,9 +1,9 @@
 click==7.1.2
-google-api-core==1.20.1
+google-api-core==1.21.0
```

Comment:
Not sure why this is showing up in the diff, but this is already `1.21.0` in master.

https://github.com/googleapis/gapic-generator-python/blob/ed244b31bd885c372a65df8e14c5759e6b5a16a4/requirements.txt#L2